go.mod

```diff
@@ -0,0 +1,178 @@
+module helm.sh/helm
+
+go 1.12
+
+require (
+	cloud.google.com/go v0.34.0 // indirect
+	contrib.go.opencensus.io/exporter/ocagent v0.2.0 // indirect
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
+	github.com/Azure/go-autorest v11.2.8+incompatible // indirect
+	github.com/BurntSushi/toml v0.3.1
```

Comment:
What do you think about moving all the direct dependencies of Helm to the top of the `require` file, `goimports` style (separated by an empty newline)? That way we can clearly separate direct and indirect dependencies. We'd still want to keep them in alphabetical order, of course. :)

Kinda like this:

```
require (
  github.com/BurntSushi/toml v0.3.1
  github.com/Masterminds/semver v1.4.2
  github.com/Masterminds/vcs v1.13.0
  ...

  cloud.google.com/go v0.34.0 // indirect
  contrib.go.opencensus.io/exporter/ocagent v0.2.0 // indirect
)
```

Reply:
If you notice, the dependencies are listed in alpha (ascii) sorted order. If I were to run a command like:

```
$ go mod -require github.com/Masterminds/semver@v1.4.2
```

The command would rewrite the file and put it back in alpha order. To control the ordering means we would need to manually manage the file all the time.

I like the idea of ordering it our way but I don't know that I want to tell people they can't use `go mod edit` to manage it.

Reply:
bleh. :( okay. I'm not too familiar with `go mod` yet. Thanks for the heads up!